### PR TITLE
ffuf: init at 1.1.0

### PR DESCRIPTION
**Pull request by htr**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/ffuf/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchFromGitHub {
+    owner = "ffuf";
+    repo = "ffuf";
+    rev = "v${version}";
+    sha256 = "1jb2x0ybcb9zkqm7flpmr0hd3171xvnn6kxmfcgds4x8l9fbmxnr";
+  };
+
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
+  doCheck = false;
```
Comment (catern) [inline pkgs/tools/security/ffuf/default.nix:19]: Can you comment about why the tests fail?

Comment (htr) [reply pkgs/tools/security/ffuf/default.nix:19]: oh, copy-paste fail. there are no tests, there are no failures.

Comment (htr) [reply pkgs/tools/security/ffuf/default.nix:19]: ok, there are test failures after all. Investigating.

Comment (doronbehar) [reply pkgs/tools/security/ffuf/default.nix:19]: @htr you don't have to comment exactly why the tests fail - you can write something simple as `# tests fail`.

**Comment by SuperSandro2000** [comment]
@htr please resolve the merge conflict.

**Comment by htr** [comment]
closing, ffuf has been merged since then

**Pull request closed without merging**